react-components/stories/Architecture.stories.tsx

```diff
@@ -96,6 +98,20 @@ export const Main: Story = {
   }
 };
 
+function StoryContent({ addModelOptions }: { addModelOptions: AddModelOptions }): ReactElement {
+  const renderTarget = useRenderTarget();
+  return (
+    <>
+      <CadModelContainer
+        addModelOptions={addModelOptions}
+        onLoad={(_model: CogniteCadModel) => {
+          renderTarget.onStartup();
+        }}
+      />
```

Comment:
Just so I understand this correctly - is `onStartup` intended to be called by the application when the viewer content has been loaded?